css/main.css

```diff
@@ -1,3 +1,243 @@
+}
+
+/* Start css for top photo  */
+.image-main {
+    background: url(../image/head.jpg) no-repeat;
+    background-size: cover;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+h1{
+    margin-top: initial;
+    margin-left: 300px;
+    color: rgb(12, 11, 11);
+    font-size: 2em;
+    margin: 0em;
+}
+
+.button {
+    margin-top: 3vh;
+    height: 30px;
+    background-color: rgb(255, 74, 42);
+    color: white;
+    border: 0vh;
+    padding: 0px 40px;
+}
+
+/*  list photos which is from gallery */
+
+  .container {
+    max-width: 100%;
+    padding-right: 15px;
+    padding-left: 15px;
+    margin-right: auto;
+    margin-left: auto;
+}
+
+.row {
+    display: flex;
+    flex-wrap: wrap;
+    margin-right: -15px;
+    margin-left: -15px;
+  }
+  div.gallery {
+    margin-top: 5mm;
+    margin: 5px;
+    border: 5px solid rgba(248, 246, 246, 0.13);
+    float: left;
+  }
+  
+  div.gallery:hover {
+    border: 1px solid rgb(75, 70, 70);
+  }
+  
+  div.gallery img {
+    width: 100%;
+    height: auto;
+  }
+  
+  div.desc {
+    padding: 15px;          
+    text-align: center;
+  }
+/* FOOTER */
+.menu li{
+color : white;
+
+}
+.footer{
+    width: 100%;
+    margin: 15vh auto 0 auto;
+    height: 1005;
+    background-color: rgb(200, 199, 209);
+}
+.row1 {
+    display: flex;
+    flex-wrap: wrap;
+    margin-top: 10px;
+    margin-right: 15px;
+    margin-left: 15px;
+  }
+.footer__title{
+    text-align: center;
+    font-size: 16px;
+    margin-top: 20px;
+}
+.footer__links{
+    width: 60vw;
+    margin: 0 auto;
+    height: 80px;
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+}
+.footer__twitter{
+    color: #55ACEE;
+}
+.footer__facebook{
+    color: #4C66A4;
+}
+.footer__instagram {
+    color: #3F729B;
+}
+.footer__icon{
+    display: flex;
+    width: 40px;
+    justify-content: center;
+    align-items: center;
+    border-radius: 50%;
+    padding: 10px;
+    margin: 10px;
+    border: 1px sol
+}
+.footer__icon:hover{
+    cursor: pointer;
+}
+.footer__text{
+    text-align: center;
+    font-size: 0.9rem;
+    font-weight: 300;
+    color: #a0c5ab;
+}
+/* grid.css part */
+@media (min-width: 576px) {
+    /* sm */
+    .container {
+      max-width: 540px;
+    }
+    .sm-col-11 {
+        width: 91.666666%;
+      }
+      div.gallery {
+        margin-top: 5mm;
+       align-items: center;
+        border: 5px solid rgba(248, 246, 246, 0.13);
+        float: left;
+      }
+}
+@media (min-width: 768px) {
+    /* md */
+    .container {
+      max-width: 720px;
+    }
+    .md-col-6 {
+        width: 50%;
+      }
+}
+@media (min-width: 992px) {
+    /* lg */
+    .container {
+      max-width: 960px;
+    }
+    .lg-col-4 {
+        width: 33.333333%;
+      }
+}
+@media (min-width: 1200px) {
+    /* xl */
+    .container {
+      max-width: 1140px;
+    }
+    .xl-col-3 {
+        width: 25%;
```

Comment:
You shouldn't be writing these styles @Erwin6997 - they are already pre-written for you inside grid.css which was provided for you in the project. 
All the container`, `row` and `col-...` classes are inside grid.css and should be imported into your HTML `head` just like your own main.css.

Please make these changes and revisit how your page flows 🙂